pkg/restapi/vc/operation/operations.go

```diff
@@ -241,7 +241,7 @@ func (o *Operation) storeVCHandler(rw http.ResponseWriter, req *http.Request) {
 	doc := operation.StructuredDocument{}
 	doc.Content = make(map[string]interface{})
 	doc.Content["message"] = data
```

Comment:
What is the data structure inside edv

Reply:
// StructuredDocument represents a Structured Document. For use with an EDV.
type StructuredDocument struct {
	ID      string                 `json:"id"`
	Meta    map[string]interface{} `json:"meta"`
	Content map[string]interface{} `json:"content"`
}